libvast/include/vast/system/actors.hpp

```diff
@@ -143,7 +143,7 @@ using evaluator_actor = typed_actor_fwd<
 /// The INDEXER actor interface.
```

Comment:
GitHub won't let me comment outside of the current diff, but the `receiver_actor` further up in this file still introduces a message handler without an atom:

```
template <class T, class... Ts>
using receiver_actor = typename typed_actor_fwd<
  // Add a new source.
typename caf::reacts_to<T, Ts...>>::unwrap;
```

Reply:
Hmm should it be enforced for the receiver_actor to have an atom? Or the users for receiver_actor should pass the atom?
Any suggestion for atom example for our use cases?

Reply:
I'm not sure I understand the distinction between the two cases, I was thinking of something like this:

```
template <class T, class... Ts>
using receiver_actor = typename typed_actor_fwd<
  // Add a new source.
typename caf::reacts_to<atom::receive, T, Ts...>>::unwrap;
```

Reply:
Perfect, i was thinking about the same :)